hack/verify-codegen.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/sh
 
 if [ "$IS_CONTAINER" != "" ]; then
-  set -x
+  set -xe
   go generate ./pkg/types/installconfig.go
-  go generate ./pkg/rhcos/ami.go
-  set +x
+  # See https://github.com/openshift/installer/pull/5447#discussion_r762340594
+  # go generate ./pkg/rhcos/ami.go
   git diff --exit-code
+  set +ex
```

Comment:
Let's leave this up above the `git diff` as it was.

Reply:
oops. yup.